### PR TITLE
Updated image version

### PR DESCRIPTION
**Pull request by ssgUPC**
@cjdcordeiro and @fcarp10  if possible then, please don't remove the branch after merging it with the master branch.

### CODE REVIEW

File: docker-compose/docker-compose.yml [hunk 1]
```diff
@@ -303,7 +303,7 @@ services:
      healthcheck:
        start_period: 60s
        retries: 5
-       test: curl --fail -s http://localhost:46070/api/v1/resource-management/categorization/start/ || exit 1
+       test: curl --fail -s http://localhost:46070/api/v1/resource-management/categorization/run/ || exit 1
```
Comment (cjdcordeiro) [inline docker-compose/docker-compose.yml]: aren't you triggering the categorization with this command?

Comment (ssgUPC) [reply docker-compose/docker-compose.yml]: Actually, for doing the health check I created another URL (http://localhost:46070/api/v1/resource-management/categorization/run/)....so, I'm triggering there to check the res-cat is up or down...

Comment (ALEJANDROJ19) [reply docker-compose/docker-compose.yml]: The name is quite confusing, but the /start endpoint is the one triggering categorization.

**Comment by cjdcordeiro** [comment]
is there a strong reason why not to delete the branch?

**Comment by ssgUPC** [comment]
> is there a strong reason why not to delete the branch?

Actually, last time I updated the health check for the res-cat service, but it was re-edited that's why if you kindly keep the branch open and merge it with master branch then it would be really helpful for me.

**Comment by cjdcordeiro** [comment]
but that's why you should not rely on your local branch. If someone changed it the last it, it's because that changed is meant to be kept. If you rely on an old outdated branch then you'll be creating PRs that either conflict with master or overwrite intermediary changes

**Comment by ssgUPC** [comment]
@cjdcordeiro okay, I'll always update my local branch with the master one....so then I guess I can avoid the conflict.

**Pull request merged**